pkg/ddl/ddl_worker.go

```diff
@@ -255,6 +260,19 @@ func (d *ddl) addBatchDDLJobs(tasks []*limitJobTask) {
 	}
 }
 
+// addBatchDDLJobsV2 gets global job IDs and delivery the DDL jobs to local TiDB
+func (d *ddl) addBatchDDLJobsV2(tasks []*limitJobTask) {
+	err := d.addBatchDDLJobs(tasks)
```

Comment:
Do we need to add the `addBatchDDLJobs2Queue` logical process? 
About the following scenarios: at < v6.2 TiDB rolling upgrade to the current v2 version, it may be < v6.2 TiDB is the owner.

Reply:
[4bb62d1](https://github.com/pingcap/tidb/pull/49687/commits/4bb62d16f976398fe962fb3c510a61069dca6769)